build-system/app.js

```diff
@@ -1079,10 +1081,17 @@ app.get('/dist/ww(.max)?.js', (req, res) => {
  * @param {string} file
  * @param {string=} hostName
  * @param {boolean=} inabox
+ * @param {boolean=} storyV1
  */
-function replaceUrls(mode, file, hostName, inabox) {
+function replaceUrls(mode, file, hostName, inabox, storyV1) {
   hostName = hostName || '';
   if (mode == 'default') {
+    // TODO:(ccordry) remove this when story 0.1 is deprecated
+    if (storyV1) {
+      file = file.replace(
+          /https:\/\/cdn\.ampproject\.org\/v0\/amp-story-0\.1\.js/g,
+          hostName + '/dist/v0/amp-story-1.0.max.js');
```

Comment:
Just curious: what's the ``.max``?

Reply:
When gulp builds the non-production binaries it adds the max to the file name. On prod they are just amp-story-1.0.js